.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,36 @@
+name: Run end-to-end tests on Gadi
+on:
+  workflow_dispatch:
+    inputs:
+      release_version:
+        description: 'Release version'     
+        required: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        ### Latest at time of writing
+        uses: actions/checkout@v4.2.2
```

Comment:
I would just assume we have a copy on Gadi for now. You can delete the checkout and the rsync.